test/color_laws.jl

```diff
@@ -286,3 +287,41 @@ end
         @test ustrip.(reddening) ≈ ref_values[rv] rtol = 0.016
     end
 end
+
+@testset "F19" begin
+    # x values from Fitzpatrick et al. (2000) Table 3
+
+    x_inv_microns = [1.0, 1.5, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0]
+    wave = 1e4 ./ x_inv_microns
+
+    # convert from E(x-V)/E(B-V) to A(x)/A(V)
+    ref_values = Dict(
+        3.1 => @. ([-1.757, -0.629, 0.438, 2.090, 4.139, 5.704, 4.904, 5.684, 7.150] + 3.1) / 3.1
+    )
+
+    # test defaults
+    @test F19().(wave) ≈ ref_values[3.1] rtol = 0.016
+
+    for rv in keys(ref_values)
+        law = F19(Rv = rv)
+        output = @inferred map(law, wave)
+        @test output ≈ ref_values[rv] rtol = 0.016
+
+        bad_waves = [100, 4e4]
+        @test @inferred(map(law, bad_waves)) == zeros(length(bad_waves))
+        @test_throws ErrorException f19_invum(aa_to_invum(bad_waves[1]), rv)
+        @test_throws ErrorException f19_invum(aa_to_invum(bad_waves[2]), rv)
+
+        # uncertainties
+        noise = rand(length(wave)) .* 0.01
+        wave_unc = wave .± noise
+        reddening = map(w -> @uncertain(law(w)), wave_unc)
```

Comment:
Do you need to use `@uncertain` also here?

Reply:
I think so, since this model calls Dierckx.jl's wrapping of the Fortran library that's doing the interpolation, like in F99 I believe.

Reply:
Ok, I asked for confirmation :slightly_smiling_face:

Reply:
I just realized it would have been perfect if Fitzpatrick et al. created interpolated models in 1900/1995. The we would have F90/F95 models calling F90/F95!

Reply:
Ah yes, Fitzpatrick '00, making use of the many mid-infrared space telescopes we had in 1900 😁

Reply:
lol, just edited